gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/MysqlUserQuotaManager.java

```diff
@@ -80,18 +87,21 @@ public void init(Collection<Dag<JobExecutionPlan>> dags) {
 
   @Override
   public void checkQuota(Collection<Dag.DagNode<JobExecutionPlan>> dagNodes) throws IOException {
-    try (Connection connection = this.quotaStore.dataSource.getConnection()) {
+    try (Connection connection = this.quotaStore.dataSource.getConnection();
+        Timer.Context context = metricContext.timer(RuntimeMetrics.GOBBLIN_MYSQL_QUOTA_MANAGER_CHECK_QUOTA_TIME).time()) {
```

Comment:
when do we call stop for this?

Reply:
This is in try clause, when this try section finish, the context will close automatically